### ISSUE TITLE
The customise option in properties is not visable for librarys

### ISSUE THREAD

**Issue by Josh-65** [files-community/Files#9445]
### Description

File explorer allows changing the icon witch files use to do but now doesnt show the section on the properties window

### Expected behavior

Customise section like for folders should show

### Files Version

2.3.0.0

### Windows Version

10.0.19043.1685

### Relevant Assets/Logs

Not relevant

**Comment by yaichenbaum** [comment]
@Josh-65 are you sure the icon change takes affect in File Explorer?

**Comment by Josh-65** [comment]
yep
![image](https://user-images.githubusercontent.com/26977075/174688048-1c3aecb4-d42f-415a-ad29-1debd7ad5351.png)
![image](https://user-images.githubusercontent.com/26977075/174688066-0c754edf-9946-49e7-aeae-f10e3b73696a.png)

**Comment by Josh65-2201** [comment]
Asking @yair100 for approval.

**Comment by yaichenbaum** [comment]
What does the properties window look like in Files?

**Comment by Josh-65** [comment]
![image](https://user-images.githubusercontent.com/26977075/195657896-04e3dcfe-417b-437f-ae64-2de8c076d442.png)

**Comment by yaichenbaum** [comment]
Would it make sense to add the options to the general page like in File Explorer or should we keep a separate tab for consistency?

**Comment by Josh-65** [comment]
I would expect it to be separate really. To keep the location and UI for setting the icon consistent. However, I'm not sure if opening a .dll and selecting an icon from it is possible since File explorer doesn't seem to allow that.

**Comment by 0x5bfa** [comment]
@Josh65-2201 Is this still an issue for you? And could you tell me how to get to the library folder location?

**Comment by Josh65-2201** [comment]
Yes, you can right click the sidebar and enable libraries > Right click one > Open properties

**Comment by 0x5bfa** [comment]
Thanks!
